packages/contracts-bedrock/package.json

```diff
@@ -11,25 +11,29 @@
   "scripts": {
     "bindings": "pnpm bindings:ts && pnpm bindings:go",
     "bindings:ts": "nx generate @eth-optimism/contracts-ts",
-    "bindings:go": "cd ../../op-bindings && make",
-    "build": "forge build",
+    "bindings:go:no-build": "cd ../../op-bindings && make binding-build",
+    "bindings:go": "pnpm clean && pnpm build && bindings:go:no-build",
```

Comment:
The result of calling `bindings:go` should be identical before and after this change, ie. it  forces a rebuild. 

See the relevant makefile section:

https://github.com/ethereum-optimism/optimism/blob/ded81369125cac37bcd76e53703627a46df5e08f/op-bindings/Makefile#L13-L27